scripts/whatsapp-stage1b-cert/generate_fixtures.py

```
@@ -0,0 +1,200 @@
+    c = canvas.Canvas(str(path), pagesize=A4)
+    y = 800
+    for line in lines:
+        c.drawString(72, y, line)
+        y -= 24
+    c.save()
+
+
+def save_audio(path: Path, text: str) -> bool:
+    wav = path.with_suffix(".wav")
+    try:
+        subprocess.run(
```

Ruff (0.16.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)